crm_properties.go

```diff
@@ -0,0 +1,107 @@
+package hubspot
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	crmPropertiesPath = "properties"
+)
+
+type CrmPropertiesList struct {
+	Results []CrmProperty
+}
+
+type CrmProperty struct {
+	Calculated       bool   `json:"calculated"`
+	Description      string `json:"description"`
+	DisplayOrder     int    `json:"displayOrder"`
+	ExternalOptions  bool   `json:"externalOptions"`
+	FieldType        string `json:"fieldType"`
+	FormField        bool   `json:"formField"`
+	GroupName        string `json:"groupName"`
+	HasUniqueValue   bool   `json:"hasUniqueValue"`
+	Hidden           bool   `json:"hidden"`
+	HubspotDefined   bool   `json:"hubspotDefined"`
+	Label            string `json:"label"`
+	ModificationMeta CrmPropertyModificationMeta
+	Name             string `json:"name"`
+	Options          []CrmPropertyOptions
```

Comment:
How about using a pointer?
```suggestion
	Options          []*CrmPropertyOptions
```

Reply:
done